fuseml/questions.yaml

```diff
@@ -8,3 +8,24 @@ questions:
   type: string
   label: Path to custom repository with FuseML extensions
   group: "Global Options"
+- variable: extensions.mlflow
+  default: false
+  required: false
+  type: boolean
+  label: Install MLFlow extension
```

Comment:
you could make it say just "Install MLFlow", I think it relays the message better

Reply:
OK